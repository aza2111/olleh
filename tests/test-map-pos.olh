//test-map-pos: testing basic map functionality

map<String> x;
x = fresh map(String);

x.set("hello", 3);
print x;
print x.get("hello"); /*3*/
print x.get("asdf"); //undefined?

map<char> y;
y = {};
print y;
y = {'x' -> 4, 'y' -> 45};
print y;
