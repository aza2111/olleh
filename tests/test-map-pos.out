{"hello" -> 3}
3
3
{'y' -> 45, 'x' -> 4}
